compliance/disallowed_images/README.md

```diff
@@ -0,0 +1,39 @@
+# Disallowed Cloud Images
+
+## What it does
+
+This policy checks all running instances for disallowed cloud images. The user is given the option to Terminate the instance after approval.
+
+## Functional Details
+
+The policy leverages the CMP API to check all instances not using the provided list of cloud image resource_uids. Running instance states include any instance with state: running, operational and provisioned. Found instances are terminated after user approval.
+
+### Input Parameters
+
+- *Email addresses to notify* - Email addresses of the recipients you wish to notify when new incidents are created.
+- *Exclusion Tag* - List of tags that will exclude Servers from being evaluated by this policy. Multiple tags are evaluated as an 'OR' condition. Tag must be of the format 'namespace:predicate=value'. Example: 'rs_agent:type=right_link_lite,rs_monitoring:state=auth'.
```

Comment:
keep terminology consistent.  use **instances** instead of **servers**.

Reply:
Updated file